it/endpoints/LookupControllerISpec.scala

```diff
@@ -42,14 +55,13 @@ class LookupControllerISpec extends IntegrationBaseSpec {
 
   "Calling the mtd lookup endpoint" when {
 
-    val nino = "AA123456A"
-
     "the user is authorised" should {
 
       "return 200" in new Test {
+
         override def setupStubs(): StubMapping = {
-          AuthStub.authorised()
-          BusinessDetailsStub.getMtdId(Json.obj("mtdbsa" -> "1234567890").toString(), nino, Status.OK)
+          AuthStub.authorised() // registration/business-details/nino/AA123456A
+          DownstreamStub.onSuccess(DownstreamStub.GET, s"/registration/business-details/nino/$nino", Status.OK, responseJson)
         }
 
         val response: WSResponse = await(request(nino).get())
```

Comment:
Although it doesn't check it currently, we ought to check that we get the right body:
```  
        response.json shouldBe Json.obj("mtdbsa" -> mtdId)
```